db/organizations/airfranceklm.eno

```diff
@@ -0,0 +1,9 @@
+name: Air France-KLM Group
+website_url: https://www.airfranceklm.com/en
+privacy_policy_url: https://wwws.airfrance.fr/en/information/legal/edito-psc
+privacy_contact:
+country:
```

Comment:
```suggestion
privacy_contact: mail.data.protection@airfrance.fr
country: FR
```

Reply:
AirFrance and KLM are one company now (sharing Flying Blue program) but I'm not sure about the nationality. Would it be sufficient to set the country as FR?

Reply:
https://en.wikipedia.org/wiki/Air_France-KLM

Headquartered in Paris, France.

Reply:
Makes sense!